.env.production.example

```
@@ -14,6 +14,17 @@ RECEIPT_OCR_TARGET_MAX_BYTES=307200
 RECEIPT_OCR_JOB_TTL_HOURS=24
 RECEIPT_OCR_JOB_MAX_ATTEMPTS=2
 RECEIPT_OCR_QUEUE_BATCH_SIZE=1
+API_RATE_LIMIT_WINDOW_MS=1000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_RATE_LIMIT_WINDOW_MS key should go before the DATABASE_URL key

(UnorderedKey)